coalib/bears/requirements/JuliaRequirement.py

```diff
@@ -29,3 +29,22 @@ def __init__(self, package, version="", flag=""):
         """
         PackageRequirement.__init__(self, 'julia', package, version)
         self.flag = flag
+
+    def install_command(self):
+        """
+       Creates the installation command for the instance of the class.
+
+       >>> JuliaRequirement('("Pkg.add(\"Lint\")', '' , '-u' ).install_command()
+       'julia -e "Pkg.add(\"Lint\")"'
+
+       :param return: A string with the installation command.
+       """
+        return "julia get {} {}".format(self.flag, self.package)
```

Comment:
this also needs work, you need the doctest to pass ;/ this is not how its going to pass